sql/src/main/java/io/crate/metadata/cluster/OpenTableClusterStateTaskExecutor.java

```diff
@@ -97,15 +97,18 @@ protected ClusterState execute(ClusterState currentState, OpenCloseTableOrPartit
             mdBuilder.put(updateOpenCloseOnPartitionTemplate(templateMetaData, true));
         }
 
+        // The MetaData will always be overridden (and not merged!) when applying it on a cluster state builder.
```

Comment:
pls document the fact that the clusterstate is already modified in the ClusterStateModifier interface. is this for any method in that interface?